pkg/lib/serverTime.js

```diff
@@ -321,13 +316,166 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230113-125341-c8c3ecb6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)